tools/replay-viewer/style.css

```
@@ -0,0 +1,564 @@
+
+body {
+  min-height: 100vh;
+}
+
+.app-shell {
+  min-height: 100vh;
+}
+
+button,
+select,
+.file-input__button {
```

Stylelint (17.11.0)

[error] Expected class selector ".file-input__button" to be kebab-case (selector-class-pattern)

(selector-class-pattern)